crates/astria-composer/src/searcher/executor.rs

```diff
@@ -50,8 +50,10 @@ use tracing::{
 pub(super) struct Executor {
     // The status of this executor
     status: watch::Sender<Status>,
-    // Channel for receving bundles to pack, sign, and submit
+    // Channel receiver for bundles to pack, sign, and submit
     executor_rx: mpsc::Receiver<Vec<Action>>,
+    // Channel sender for resubmitting transactions that failed submission due to failed nonce
+    transaction_resubmission_tx: mpsc::Sender<Vec<Action>>,
```

Comment:
it appears this is the sender counterpart of `executor_rx`?  maybe add a comment for that?